packages/material-ui-system/src/createTheme/createTheme.d.ts

```diff
@@ -15,7 +14,7 @@ export interface ThemeOptions {
   palette?: Record<string, any>;
   shadows?: any;
   spacing?: SpacingOptions;
-  transitions?: TransitionsOptions;
+  transitions?: any;
   components?: Record<string, any>;
   typography?: any;
```

Comment:
`unknown` for new types.

Reply:
Done with https://github.com/mui-org/material-ui/pull/26490/commits/b8f0966915b93174d17e903ac9283d83810058d3.